dispatcher.py

```diff
@@ -6,6 +6,8 @@
 import functools
```

Comment:
Regarding commit message:

1. 'fix multiprocessing module bug with serialization' is not what we're doing here. We set 'fork' start method (and the change re lambda elimination becomes unnecessary and should be removed).
2. Why `Part of #265`? It seems, it'll be resolved by using the 'fork' start method. And we unlikely will investigate how to adapt code to be ready for using 'spawn'.
3. Details are desirable:
    - 'spawn' becomes default on Mac OS and we need to set 'fork' explicitly (this can be included right into the code).
    - Why 'spawn' method does not work for us (you can use my investigation from another comment or, if you know another critical problem, use it to advocate the change); in fact, this also can be included right into the code.

Reply:
1. Agree. Initially there was another attempt to fix issue.
2. convert to 'Fixes"
3. add wording to commit message and to source code inline

Reply:
Object serialization is not the primary problem. Code reexecution in a child process is the reason.

Reply:
I'm going to change the wording. See the [comment](https://github.com/tarantool/test-run/pull/263#discussion_r593350504) above.